syncedlyrics/providers/musixmatch.py

```diff
@@ -120,12 +128,13 @@ def get_lrc(self, search_term: str) -> Optional[str]:
             return None
         body = r.json()["message"]["body"]
         tracks = body["track_list"]
-        cmp_key = lambda t: f"{t['track']['track_name']} {t['track']['artist_name']}"
+        def cmp_key(t): return f"{t['track']['track_name']} {t['track']['artist_name']}"
```

Comment:
Any reason why you replaced lambda?

Reply:
Oh nothing in particular I think it got replaced automatically on save in vscode because pep8 conventions want this or something, I personally don't care, I can change it back

Reply:
Alright, the project follows formatting of `black` module. Can you also run `poetry run black synceldyrics` before merging.